src/logging.c

```diff
@@ -251,13 +251,30 @@ rstrnt_on_log_uploaded (SoupSession *session,
     g_mapped_file_unref (user_data);
 }
 
+static void
+rstrnt_flush_log_data (const RstrntLogData *log_data,
+                       GCancellable        *cancellable)
+{
+    GOutputStream      *stream;
+    g_autofree gchar   *path = NULL;
+    g_autoptr (GError)  error = NULL;
+
+    stream = G_OUTPUT_STREAM (log_data->output_stream);
+
+    if (g_output_stream_flush (stream, cancellable, &error)
+        || G_IO_ERROR_CANCELLED == error->code)
+        return;
+
+    path = g_file_get_path (log_data->file);
```

Comment:
API doc says to g_free 'path' when no longer needed.

Reply:
Line 259, `g_autofree gchar   *path = NULL;`

And valgrind checks would have complained about that.

Reply:
I looked for g_autofree but saw log_path definition instead.We're good.

Reply:
> I looked for g_autofree but saw log_path definition instead.We're good.

There is no `log_path` in this function.